### PR TITLE
Update docker-compose references to "docker compose"

### PR DESCRIPTION
**Pull request by Proph151Music**
Addresses #6

### CODE REVIEW

File: scripts/hydra [hunk 1]
```diff
@@ -193,7 +193,7 @@ install() {
   echo "Installing hydra ..."
   echo "Installing Framework..."
   create_template project $PROJECT_NAME
-
+  chmod -R 777 infra/docker/monitoring
```
Comment (IPadawans) [inline scripts/hydra:196]: We don't need the 777 on CHMOD at this point, you can update to:
`chmod -R a+rwx ...your_directory`

**Review by IPadawans — CHANGES_REQUESTED** [review]

**Comment by IPadawans** [comment]
@Proph151Music Take a look at my comment in your PR and update your branch according to `develop` branch

File: scripts/hydra (hunk 1 above)
Comment (IPadawans) [reply scripts/hydra]: Yeah, and the rwx is equivalent to chmod 755, I don't know exactly why haha but they are different

**Comment by Proph151Music** [comment]
The other method I tried was creating my own p12 files by exporting the details:
```export CL_KEYSTORE="techware-genesis.p12"
export CL_KEYALIAS="techware-genesis"
export CL_PASSWORD="MyPassword"```

and then running this command:
**java -jar cl-keytool.jar generate**

Then I edit the .env and overwrite the files in source/p12-files.

Both methods end with the same results.  So maybe I am doing something wrong?

**Pull request merged**